apps/web/src/common/modules/navigations/top-bar/modules/gnb-search-clone/modules/gnb-search-dropdown/modules/TopBarSearchWorkspaceFilter.vue

```diff
@@ -16,12 +16,22 @@ const workspaceStoreState = userWorkspaceStore.$state;
 const topBarSearchStore = useTopBarSearchStore();
 
 const state = reactive({
+    currentWorkspaceId: computed(() => workspaceStoreState.getters.currentWorkspaceId),
     workspaceList: computed<WorkspaceModel[]>(() => [...workspaceStoreState.getters.workspaceList]),
-    workspaces: computed(() => state.workspaceList.map((workspace) => ({
-        label: workspace.name,
-        value: workspace.workspace_id,
-        tags: workspace.tags,
-    } as { label: string, value: string, tags: { theme: string } | undefined }))),
+    workspaces: computed(() => {
+        const workspaceList = state.workspaceList.map((workspace) => ({
+            label: workspace.name,
+            value: workspace.workspace_id,
+            tags: workspace.tags,
+        } as { label: string, value: string, tags: { theme: string } | undefined }));
+        // 현재 워크스페이스를 가장 상단에 위치시키기 위해 정렬
```

Comment:
🙊

Reply:
I can't believe you're giving me a review during your annual leave... Thank you!